app/assets/stylesheets/global.scss

```diff
@@ -1,3 +1,40 @@
+@font-face {
+  font-family: "cityscape";
+  src: asset-url("cityscape");
+}
+
+@font-face {
+  font-family: "lethal_league";
+  src: asset-url("Lethal_League");
+}
+
+@font-face {
+  font-family: "will-harris-reg";
+  src: asset-url("Will-Harris-Regular");
+}
+
+@font-face {
+  font-family: "hometown";
+  src: asset-url("Hometown");
+}
+
+@font-face {
+  font-family: "assassin";
+  src: asset-url("Assassin");
+}
+
+@font-face {
+  font-family: "romance";
+  src: asset-url("romance");
+}
+
+@font-face {
```

Comment:
This is just resetting @font-face over and over. So only the last one is actually saving. The rest is unnecessary code. I assume this is not the behavior you want